<!doctype html><html lang=en class=no-js> <head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta name=description content="Some bioinformatics knowledge gathered by Jacques Dainat"><meta name=author content="Jacques Dainat"><link href=https://juke34.github.io/awesome-genome-annotation/annotation/structural/annotation_tools_genome/ rel=canonical><link rel=prev href=../../..><link href=../annotation_tools_mitome/ rel=next><link rel=icon href=../../../assets/images/favicon.png><meta name=generator content="mkdocs-1.6.1, mkdocs-material-9.6.15"><title>Genome - Awesome Genome Annotation Tools</title><link rel=stylesheet href=../../../assets/stylesheets/main.342714a4.min.css><link rel=stylesheet href=../../../assets/stylesheets/palette.06af60db.min.css><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback"><style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style><link rel=stylesheet href=../../../stylesheets/extra.css><script>__md_scope=new URL("../../..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script></head> <body dir=ltr data-md-color-scheme=default data-md-color-primary=indigo data-md-color-accent=indigo> <input class=md-toggle data-md-toggle=drawer type=checkbox id=__drawer autocomplete=off> <input class=md-toggle data-md-toggle=search type=checkbox id=__search autocomplete=off> <label class=md-overlay for=__drawer></label> <div data-md-component=skip> <a href=#list-of-genome-annotation-tools class=md-skip> Skip to content </a> </div> <div data-md-component=announce> </div> <header class=md-header data-md-component=header> <nav class="md-header__inner md-grid" aria-label=Header> <a href=../../.. title="Awesome Genome Annotation Tools" class="md-header__button md-logo" aria-label="Awesome Genome Annotation Tools" data-md-component=logo> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg> </a> <label class="md-header__button md-icon" for=__drawer> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg> </label> <div class=md-header__title data-md-component=header-title> <div class=md-header__ellipsis> <div class=md-header__topic> <span class=md-ellipsis> Awesome Genome Annotation Tools </span> </div> <div class=md-header__topic data-md-component=header-topic> <span class=md-ellipsis> Genome </span> </div> </div> </div> <form class=md-header__option data-md-component=palette> <input class=md-option data-md-color-media data-md-color-scheme=default data-md-color-primary=indigo data-md-color-accent=indigo aria-label="Switch to dark mode" type=radio name=__palette id=__palette_0> <label class="md-header__button md-icon" title="Switch to dark mode" for=__palette_1 hidden> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M12 8a4 4 0 0 0-4 4 4 4 0 0 0 4 4 4 4 0 0 0 4-4 4 4 0 0 0-4-4m0 10a6 6 0 0 1-6-6 6 6 0 0 1 6-6 6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg> </label> <input class=md-option data-md-color-media data-md-color-scheme=slate data-md-color-primary=indigo data-md-color-accent=indigo aria-label="Switch to light mode" type=radio name=__palette id=__palette_1> <label class="md-header__button md-icon" title="Switch to light mode" for=__palette_0 hidden> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M12 18c-.89 0-1.74-.2-2.5-.55C11.56 16.5 13 14.42 13 12s-1.44-4.5-3.5-5.45C10.26 6.2 11.11 6 12 6a6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg> </label> </form> <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script> <label class="md-header__button md-icon" for=__search> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg> </label> <div class=md-search data-md-component=search role=dialog> <label class=md-search__overlay for=__search></label> <div class=md-search__inner role=search> <form class=md-search__form name=search> <input type=text class=md-search__input name=query aria-label=Search placeholder=Search autocapitalize=off autocorrect=off autocomplete=off spellcheck=false data-md-component=search-query required> <label class="md-search__icon md-icon" for=__search> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg> </label> <nav class=md-search__options aria-label=Search> <a href=javascript:void(0) class="md-search__icon md-icon" title=Share aria-label=Share data-clipboard data-clipboard-text data-md-component=search-share tabindex=-1> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7s-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91s2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08"/></svg> </a> <button type=reset class="md-search__icon md-icon" title=Clear aria-label=Clear tabindex=-1> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg> </button> </nav> <div class=md-search__suggest data-md-component=search-suggest></div> </form> <div class=md-search__output> <div class=md-search__scrollwrap tabindex=0 data-md-scrollfix> <div class=md-search-result data-md-component=search-result> <div class=md-search-result__meta> Initializing search </div> <ol class=md-search-result__list role=presentation></ol> </div> </div> </div> </div> </div> <div class=md-header__source> <a href=https://github.com/juke34/awesome-genome-annotation title="Go to repository" class=md-source data-md-component=source> <div class="md-source__icon md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 448 512"><!-- Font Awesome Free 6.7.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81"/></svg> </div> <div class=md-source__repository> awesome-genome-annotation </div> </a> </div> </nav> </header> <div class=md-container data-md-component=container> <nav class=md-tabs aria-label=Tabs data-md-component=tabs> <div class=md-grid> <ul class=md-tabs__list> <li class="md-tabs__item md-tabs__item--active"> <a href=../../.. class=md-tabs__link> Awesome Genome Annotation Tools </a> </li> </ul> </div> </nav> <main class=md-main data-md-component=main> <div class="md-main__inner md-grid"> <div class="md-sidebar md-sidebar--primary" data-md-component=sidebar data-md-type=navigation> <div class=md-sidebar__scrollwrap> <div class=md-sidebar__inner> <nav class="md-nav md-nav--primary md-nav--lifted md-nav--integrated" aria-label=Navigation data-md-level=0> <label class=md-nav__title for=__drawer> <a href=../../.. title="Awesome Genome Annotation Tools" class="md-nav__button md-logo" aria-label="Awesome Genome Annotation Tools" data-md-component=logo> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg> </a> Awesome Genome Annotation Tools </label> <div class=md-nav__source> <a href=https://github.com/juke34/awesome-genome-annotation title="Go to repository" class=md-source data-md-component=source> <div class="md-source__icon md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 448 512"><!-- Font Awesome Free 6.7.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81"/></svg> </div> <div class=md-source__repository> awesome-genome-annotation </div> </a> </div> <ul class=md-nav__list data-md-scrollfix> <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested"> <input class="md-nav__toggle md-toggle " type=checkbox id=__nav_1 checked> <div class="md-nav__link md-nav__container"> <a href=../../.. class="md-nav__link "> <span class=md-ellipsis> Awesome Genome Annotation Tools </span> </a> <label class="md-nav__link " for=__nav_1 id=__nav_1_label tabindex> <span class="md-nav__icon md-icon"></span> </label> </div> <nav class=md-nav data-md-level=1 aria-labelledby=__nav_1_label aria-expanded=true> <label class=md-nav__title for=__nav_1> <span class="md-nav__icon md-icon"></span> Awesome Genome Annotation Tools </label> <ul class=md-nav__list data-md-scrollfix> <li class="md-nav__item md-nav__item--active"> <input class="md-nav__toggle md-toggle" type=checkbox id=__toc> <a href=./ class="md-nav__link md-nav__link--active"> <span class=md-ellipsis> Genome </span> </a> </li> <li class=md-nav__item> <a href=../annotation_tools_mitome/ class=md-nav__link> <span class=md-ellipsis> Mitochondrion </span> </a> </li> <li class=md-nav__item> <a href=../annotation_tools_plastome/ class=md-nav__link> <span class=md-ellipsis> Chloroplast </span> </a> </li> <li class=md-nav__item> <a href=../annotation_tools_plasmidome/ class=md-nav__link> <span class=md-ellipsis> Plasmid </span> </a> </li> </ul> </nav> </li> </ul> </nav> </div> </div> </div> <div class=md-content data-md-component=content> <article class="md-content__inner md-typeset"> <a href=https://github.com/juke34/awesome-genome-annotation/edit/main/docs/annotation/structural/annotation_tools_genome.md title="Edit this page" class="md-content__button md-icon" rel=edit> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M10 20H6V4h7v5h5v3.1l2-2V8l-6-6H6c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h4zm10.2-7c.1 0 .3.1.4.2l1.3 1.3c.2.2.2.6 0 .8l-1 1-2.1-2.1 1-1c.1-.1.2-.2.4-.2m0 3.9L14.1 23H12v-2.1l6.1-6.1z"/></svg> </a> <a href=https://github.com/juke34/awesome-genome-annotation/raw/main/docs/annotation/structural/annotation_tools_genome.md title="View source of this page" class="md-content__button md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M17 18c.56 0 1 .44 1 1s-.44 1-1 1-1-.44-1-1 .44-1 1-1m0-3c-2.73 0-5.06 1.66-6 4 .94 2.34 3.27 4 6 4s5.06-1.66 6-4c-.94-2.34-3.27-4-6-4m0 6.5a2.5 2.5 0 0 1-2.5-2.5 2.5 2.5 0 0 1 2.5-2.5 2.5 2.5 0 0 1 2.5 2.5 2.5 2.5 0 0 1-2.5 2.5M9.27 20H6V4h7v5h5v4.07c.7.08 1.36.25 2 .49V8l-6-6H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h4.5a8.2 8.2 0 0 1-1.23-2"/></svg> </a> <h1 id=list-of-genome-annotation-tools>List of genome annotation tools<a class=headerlink href=#list-of-genome-annotation-tools title="Permanent link">#</a></h1> <table> <thead> <tr> <th>year</th> <th>Tool name</th> <th>Publication</th> <th>Type</th> <th>Method</th> <th>Organism</th> <th>Comments</th> </tr> </thead> <tbody> <tr> <td>1991</td> <td>GRAIL</td> <td>E. C. Uberbacher and R. J. Mural (1991), "Locat- ing protein-coding regions in human DNAsequences by a multiple sensor-neural network approach", Proc. Natl. Acad. Sci. USA,Vol. 88, pp. 11261- 11265.<br>R. J. Mural, J. R. Einstein, X. Guan, R. C. Mann and E. C. Uberbacher(1992), "All Artificial Intelli- gence Approach to DNASequence Feature Recogni- tion", Trend in Biotechnology, 10, pp. 66 - 69.</td> <td>Ab initio (sensors + Neural network)</td> <td></td> <td></td> <td>No longer supported</td> </tr> <tr> <td>1991</td> <td>NetGene</td> <td>Brunak, S., Engelbrecht, J., and Knudsen, S. (1991). Prediction of human mRNA donor and acceptor sites from the DNA sequence. J. Mol. Biol. 220, 49–65.</td> <td>Ab initio</td> <td></td> <td></td> <td></td> </tr> <tr> <td>1992</td> <td>GeneID</td> <td>Guigo, R., Knudsen, S., Drake, N., and Smith, T. (1992), Prediction of gene structure J. Mol. Biol. 226, 141–157.</td> <td>Ab initio</td> <td>WAM, HMM, PD, AD, NN</td> <td></td> <td></td> </tr> <tr> <td>1992</td> <td>GeneID+</td> <td>Guigo, R., Knudsen, S., Drake, N., and Smith, T. (1992), Prediction of gene structure J. Mol. Biol. 226, 141–157.</td> <td>Hybrid</td> <td>WAM, HMM, PD, AD, NN</td> <td></td> <td>use information from protein sequence database searches</td> </tr> <tr> <td>1992</td> <td>SORFIND</td> <td>Hutchinson, G. B., and Hayden, M. R. (1992) Nucleic Acids Res. 20, 3453–3462.</td> <td>Abinitio</td> <td></td> <td></td> <td></td> </tr> <tr> <td>1993</td> <td>Genemark</td> <td>Borodovsky and McIninch</td> <td>Ab initio</td> <td></td> <td></td> <td></td> </tr> <tr> <td>1993</td> <td>Geneparser</td> <td>Snyder, E.E. and Stormo, G.D. 1993. Identification of coding regions in genomic DNA sequences: an application of dynamic programming and neural networks. Nucleic Acids Res. 21: 607-613.</td> <td>Ab initio</td> <td>DP combined with a neural network program</td> <td></td> <td></td> </tr> <tr> <td>1994</td> <td>GRAIL-II</td> <td>Recognizing exons in genomic sequence using GRAIL II. Xu Y, Mural R, Shah M, Uberbacher E. Genet Eng (N Y). 1994; 16():241-53.</td> <td>Ab initio</td> <td> </td> <td></td> <td></td> </tr> <tr> <td>1994</td> <td>Xpound</td> <td>Thomas,A. and Skolnick,M.H. (1994) A probabilistic model for detecting coding regions in DNA sequences. IMA J. Math. Appl. Med. Biol., 11, 149–160.</td> <td>Ab initio</td> <td> </td> <td></td> <td></td> </tr> <tr> <td>1994</td> <td>EcoParse</td> <td></td> <td>Ab initio</td> <td>HMM</td> <td>Prokaryote</td> <td></td> </tr> <tr> <td>1994</td> <td>GeneLang / GenLang</td> <td>Dong, S. and Searls, D.B. 1994. Gene structure prediction by linguistic methods. Genomics 23: 540-551.</td> <td>Ab initio</td> <td>Linguistic method HMM, PD, WAM</td> <td>Eukaryote</td> <td></td> </tr> <tr> <td>1995</td> <td>Fgeneh (Find gene in human) / GeneFinder</td> <td>Solovyev VV, Salamov AA, Lawrence CB (1995) Identification of human gene structure using linear discriminant functions and dynamic programming. Proceedings/International Conference on Intelligent Systems for Molecular Biology; ISMB International Conference on Intelligent Systems for Molecular Biology 3: 367–375</td> <td>Ab initio</td> <td>HMM, DP, LDA</td> <td>Human</td> <td>Finds single exon only</td> </tr> <tr> <td>1995</td> <td>Geneparser2</td> <td>Snyder EE, Stormo GD J Mol Biol. 1995 Apr 21; 248(1):1-18.</td> <td>Ab initio</td> <td>DP combined with a neural network program</td> <td></td> <td></td> </tr> <tr> <td>1995</td> <td>Geneparser3</td> <td>Snyder EE, Stormo GD J Mol Biol. 1995 Apr 21; 248(1):1-18.</td> <td>hybrid</td> <td>DP combined with a neural network program</td> <td></td> <td></td> </tr> <tr> <td>1996</td> <td>GeneHacker</td> <td>Yada.T , Hirosawa.M DNA Res., 3, 335-361 (1996). Syst. Mol. Biol. pp.252-260 (1996). Syst. Mol. Biol. pp.354-357 (1997).. </td> <td>ab initio</td> <td>Markov model</td> <td>Prokaryote</td> <td></td> </tr> <tr> <td>1996</td> <td>Genie</td> <td>Kulp, D.; Haussler, D.; Reese, M. G.; and Eeckman, F. H. 1996. A generalized hidden Markov model for the recognition of human genes in DNA. In D.J. States et al., ed., Proc. Conf. on Intelligent Systems in Molecular Biology, 134–142. Menlo Park, CA: AAAI Press. </td> <td>Hybrid</td> <td>GHMM + neural networks</td> <td></td> <td></td> </tr> <tr> <td>1996</td> <td>Procrustes</td> <td>Gene recognition via spliced sequence alignment. Gelfand MS, Mironov AA, Pevzner PA. Proc Natl Acad Sci U S A. 1996 Aug 20; 93(17):9061-6.</td> <td>Evidence based</td> <td></td> <td></td> <td></td> </tr> <tr> <td>1997</td> <td>Fgenes / GeneFinder</td> <td>Solovyev </td> <td>Ab initio</td> <td>HMM, DP, LDA</td> <td>Human</td> <td></td> </tr> <tr> <td>1997</td> <td>GenScan</td> <td>Burge, C. (1997). Identification of genes in human genomic DNA. Ph.D. thesis, Stanford University. ; Burge, C. &amp; Karlin, S. (1997). Prediction of complete gene structures in genomic DNA. Journal of Molecular Biology, 268,78–94</td> <td>Ab initio</td> <td>GHMM</td> <td></td> <td>GENSCAN++ is a reimplementation of GENSCAN in C++ (~2001)</td> </tr> <tr> <td>1997</td> <td>MZEF</td> <td>Identification of protein coding regions in the human genome by quadratic discriminant analysis. Zhang MQ. Proc Natl Acad Sci U S A. 1997 Jan 21; 94(2):565-8.</td> <td></td> <td>Quadratic discriminant analysis</td> <td></td> <td></td> </tr> <tr> <td>1997</td> <td>HMMGene</td> <td>Krogh A. Two methods for improving performace of a HMM and their application for gene finding. In: Gaasterland T, Karp P, Karplus K, Ouzounis C, Sander C, Valencia A, editors. The fifth international conference on intelligent Systems for Molecular Biology. CA: Menlo Park: AAAI Press; 1997. pp. 179–186.</td> <td>Ab initio</td> <td>CHMM</td> <td>Vertebrate and C. elegans</td> <td>No download version. Webserver.</td> </tr> <tr> <td>1997</td> <td>GeneWise (from Wise2 distribution)</td> <td>unplublished. Birney, E. and Durbin, R. 1997. Wise2. http://www.sanger.ac.uk/Software/Wise2.</td> <td>Evidence based</td> <td></td> <td></td> <td></td> </tr> <tr> <td>1997</td> <td>AAT (Analysis and Annotation Tool)</td> <td>Huang et al.</td> <td>Evidence based</td> <td></td> <td></td> <td>Include two paris of programs DPS/NAP and DDS/GAP</td> </tr> <tr> <td>1998</td> <td>Orpheus</td> <td>Frishman D, Mironov A, Mewes HW, Gelfand M (1998) Combining diverse evidence for gene recognition in completely sequenced bacterial genomes. Nucleic Acids Res 26:2941–2947</td> <td>abinintio + evidence</td> <td>Seed and extend</td> <td>Prokaryote / Archaea</td> <td></td> </tr> <tr> <td>1998</td> <td>SIM4</td> <td>A computer program for aligning a cDNA sequence with a genomic DNA sequence. Florea L, Hartzell G, Zhang Z, Rubin GM, Miller W. Genome Res. 1998 Sep; 8(9):967-74.</td> <td></td> <td></td> <td></td> <td></td> </tr> <tr> <td>1998</td> <td>GIN</td> <td>Y. Cai and P. Bork, “Homology-based gene prediction using neural nets, Analytical Biochemistry, vol. 265, no. 2, pp. 269–274, 1998.</td> <td>Hybrid</td> <td>NN + homology</td> <td>Vertebrate</td> <td></td> </tr> <tr> <td>1998</td> <td>GAIA</td> <td>GAIA: framework annotation of genomic sequence. Bailey LC Jr, Fischer S, Schug J, Crabtree J, Gibson M, Overton GC. Genome Res. 1998 Mar; 8(3):234-50.</td> <td>homology-based</td> <td></td> <td></td> <td></td> </tr> <tr> <td>1998</td> <td>MORGAN (Multi-frame Optimal Rule-based Gene ANalyzer)</td> <td>Salzberg S, Delcher AL, Fasman KH, Henderson J. J Comput Biol. 1998 Winter; 5(4):667-80.</td> <td>Abinitio</td> <td>DP algorithm in combination with a decision tree program</td> <td></td> <td>Hybrid tool combining decision trees with dynamic programming and signal sensor algorithm</td> </tr> <tr> <td>1998</td> <td>GeneMark.hmm</td> <td>Lukashin, A. V &amp; Borodovsky, M. GeneMark.hmm: new solutions for gene finding. Nucleic Acids Res. 26, 1107–1115 (1998).</td> <td>Ab initio</td> <td>HMM. Iteratively trains and improves the model in an unsupervised manner</td> <td>Prokaryote / Archaea</td> <td>Self training</td> </tr> <tr> <td>1998</td> <td>Glimmer</td> <td>Salzberg, S., Delcher, A., Kasif, S., and White, O. (1998b). Microbialgene identification using interpolated Markov models.Nucleic Ac-ids Res.26(2), 544 –548.</td> <td>Abinitio</td> <td>IMM</td> <td>Prokartyote + Archaea</td> <td></td> </tr> <tr> <td>1999</td> <td>Fgenesh</td> <td>Solovyev and Salamov</td> <td>HMM</td> <td></td> <td>programs that have organism-specific parameters for human, Drosophila, plants, yeast, and nematode</td> <td></td> </tr> <tr> <td>1999</td> <td>GlimmerM</td> <td>Salzberg,S.L., Pertea,M., Delcher,A.L., Gardner,M.J. and Tettelin,H. (1999) Interpolated Markov models for eukaryotic gene finding. Genomics, 59, 24–31.</td> <td>Abinito</td> <td>IMM</td> <td>Small eukaryote</td> <td>developed to find genes in the malaria parasite Plasmodium falciparum.</td> </tr> <tr> <td>1999</td> <td>Veil (the Viterbi Exon-Intron Locator)</td> <td>Finding Genes in Human DNA with a Hidden Markov Model. J. Henderson, S.L. Salzberg, and K. Fasman. This describes the VEIL system for finding genes. Journal of Computational Biology 4:2 (1997), 127-141.</td> <td>HMM</td> <td></td> <td>Eukaryote</td> <td></td> </tr> <tr> <td>1999</td> <td>CRITICA (Coding Region Identification Tool Invoking Comparative Analysis)</td> <td>Badger and Olsen. Molecular Biology and Evolution, 16(4):512-524. 1999.</td> <td>Comparative</td> <td></td> <td>Prokaryote / Archaea</td> <td>Comparative analysis is based on amino acid sequence similarity to other species</td> </tr> <tr> <td>2000</td> <td>Fgenesh+</td> <td>Salamov AA, Solovyev VV Genome Res. 2000 Apr; 10(4):516-22.; Solovyev V.V. (2007) Statistical approaches in Eukaryotic gene prediction. In Handbook of Statistical genetics (eds. Balding D., Cannings C., Bishop M.), Wiley-Interscience; 3d edition, 1616 p.</td> <td>HMM plus similar protein-based gene prediction</td> <td></td> <td></td> <td>Fgenesh+ is a variant of Fgenesh that takes into account some information about similar proteins</td> </tr> <tr> <td>2000</td> <td>Rosetta</td> <td>Batzoglou et al., 2000</td> <td>Comparative genomics</td> <td></td> <td></td> <td>Two genomes. Uses pairwise genomic alignments to find regions of homology; incorporates a splice junction and exon length model.</td> </tr> <tr> <td>2000</td> <td>CEM</td> <td>Bafna &amp; Huson, 2000</td> <td>Comparative genomics</td> <td></td> <td></td> <td>Two genomes</td> </tr> <tr> <td>2001</td> <td>GenomeScan</td> <td>Computational inference of homologous gene structures in the human genome. Yeh RF, Lim LP, Burge CB, Genome Res. 2001 May; 11(5):803-16.</td> <td>Comparative</td> <td></td> <td></td> <td></td> </tr> <tr> <td>2001</td> <td>Eugene</td> <td></td> <td>Hybrid</td> <td>Semi-Markov Conditional Random Fields / IMM, DP</td> <td>Plant</td> <td>Can be seen as a combiner because collect information about splice sites and ATG has to be done outside the program.</td> </tr> <tr> <td>2001</td> <td>Twinscan</td> <td>Ian Korf, Paul Flicek, Daniel Duan, Michael R. Brent. Bioinformatics, Volume 17, Issue suppl_1, June 2001, Pages S140–S148, https://doi.org/10.1093/bioinformatics/17.suppl_1.S140</td> <td>comparative-genomics-based</td> <td></td> <td></td> <td>Two genomes. Uses local alignments between a target genome and a reference (informant) genome to identify regions of conservation</td> </tr> <tr> <td>2001</td> <td>GeneHacker Plus</td> <td>Yada,T., Totoki,Y., Takagi,T. and Nakai,K. ( 2001 ) A novel bacterial gene‐finding system with improved accuracy in locating start codons. DNA Res. , 8 , 97 –106</td> <td>Ab initio</td> <td>HMM</td> <td>Prokaryote</td> <td></td> </tr> <tr> <td>2001</td> <td>GeneMarkS</td> <td>GeneMarkS: a self-training method for prediction of gene starts in microbial genomes. Implications for finding sequence motifs in regulatory regions. Besemer J, Lomsadze A, Borodovsky M. Nucleic Acids Res. 2001 Jun 15; 29(12):2607-18.</td> <td>Ab initio</td> <td>HMM</td> <td>Prokaryote</td> <td>Self training</td> </tr> <tr> <td>2001</td> <td>SGP-1 (Syntenic Gene Prediction)</td> <td>SGP-1: prediction and validation of homologous genes based on sequence alignments. Wiehe T, Gebauer-Jung S, Mitchell-Olds T, Guigó R. Genome Res. 2001 Sep; 11(9):1574-83.</td> <td>Comparative</td> <td></td> <td>vertebrates and plants</td> <td>Dual genomes. Uses pairwise genomic alignments to find syntenic loci; evaluates a coding and splice model in these loci.</td> </tr> <tr> <td>2001</td> <td>Spidey</td> <td>Spidey: a tool for mRNA-to-genomic alignments. Wheelan SJ, Church DM, Ostell JM. Genome Res. 2001 Nov; 11(11):1952-7.</td> <td></td> <td></td> <td></td> <td></td> </tr> <tr> <td>2002</td> <td>DOUBLESCAN</td> <td>Meyer IMM, Durbin R. 2002. Comparative ab initio prediction of gene structures using pair HMMs. Bioinformatics 18:1309–18</td> <td>comparative</td> <td>PHMM</td> <td></td> <td>Uses a pair HMM to simultaneously predict gene structures and conservation in two aligned sequences</td> </tr> <tr> <td>2002</td> <td>AGenDA (Alignment-based Gene-Detection Algorithm)</td> <td>Oliver Rinner and Burkhard Morgenstern. AGenDA: Gene Prediction by Com- parative Sequence Analysis. Silica Biology, 2:4673-4680, 2002.</td> <td>comparative</td> <td></td> <td>Eukaryote</td> <td>Based on pair-wise alignments created by CHAOS and DIALIGN</td> </tr> <tr> <td>2002</td> <td>GAZE</td> <td>Howe, K. L. et al. GAZE : A Generic Framework for the Integration of Gene-Prediction Data by Dynamic Programming. 1418–1427 (2002). doi:10.1101/gr.149502 </td> <td>Comparative / combiner</td> <td></td> <td></td> <td></td> </tr> <tr> <td>2002</td> <td>BDGF</td> <td>Shibuya T, Rigoutsos I (2002) Dictionary-driven prokaryotic gene finding. Nucleic Acids Res 30:2710–2725 </td> <td>Evidence based</td> <td></td> <td>Prokaryote / Archaea</td> <td>Classifications based on universal CDS-specific usage of short amino acid “seqlets”</td> </tr> <tr> <td>2003</td> <td>EvoGene</td> <td>Pedersen JS, Hein J. 2003. Gene finding with a hidden Markov model of genome structure and evolution. Bioinformatics 19:219–27</td> <td>Comparative / evolutionary</td> <td>Evolutionary Hidden Markov Model (EHMM)</td> <td></td> <td>Phylogenetic HMM that performs ab initio prediction of genes across a multiple-sequence alignment (more than two genomes), making use of phylogenetic information</td> </tr> <tr> <td>2003</td> <td>GeneMarkS (virus version)</td> <td>Mills R, Rozanov M, Lomsadze A, Tatusova T, Borodovsky M. Improving gene annotation of complete viral genomes. Nucleic Acids Res. 2003;31(23):7041–7055. doi: 10.1093/nar/gkg878.</td> <td>Ab initio</td> <td>HMM</td> <td>Virus</td> <td>Self training</td> </tr> <tr> <td>2003</td> <td>GeneComber</td> <td>Shah SP, McVicker GP, Mackworth AK, Rogic S, Ouellette BF. 2003. GeneComber: combining outputs of gene prediction programs for improved results. Bioinformatics 19:1296–97</td> <td>Combiner</td> <td>EUI, GI and EUI frame algorithms</td> <td></td> <td>It runs Genscan and HMMgene and combines results</td> </tr> <tr> <td>2003</td> <td>AUGUSTUS</td> <td>Stanke, M. &amp; Waack, S. Gene prediction with a hidden Markov model and a new intron submodel. Bioinformatics 19 Suppl 2, ii215–ii225 (2003).</td> <td>abinitio</td> <td>HMM</td> <td>Eukaryote</td> <td></td> </tr> <tr> <td>2003</td> <td>SLAM</td> <td>M. Alexandersson, S. Cawley, and L. Pachter. 2003. SLAM: Cross-species gene finding and alignment with a generalized pair hidden Markov model. Genome Res., 13:496-502.</td> <td>Comparative</td> <td>GPHMM (Generalized pair HMM)</td> <td>Eukaryote</td> <td>Dual genome. Treats two alignments in a symmetric way, predicting pairs of transcripts</td> </tr> <tr> <td>2003</td> <td>SGP2</td> <td>G. Parra, P. Agarwal, J.F. Abril, T. Wiehe, J.W. Fickett, and R. Guigo. 2003. Comparative gene prediction in human and mouse. Genome Res., 13:108-117</td> <td>comparative</td> <td></td> <td>Eukaryote</td> <td>Dual genome. It integrates the sequence similarity search program TBLASTX (WU-BLAST) and the ab initio gene finder GeneiD. Used by the Mouse Genome Sequencing Consortium in 2002 to annotate the mouse genome. Uses pairwise genomic alignments to find syntenic loci; evaluates a coding and splice model in these loci.</td> </tr> <tr> <td>2003</td> <td>PASA (Program to Assemble Spliced Alignments)</td> <td>Haas BJ, Delcher AL, Mount SM, Wortman JR, Smith RK Jr., et al. 2003. Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies. Nucleic Acids Res</td> <td>pipeline - combiner - evidence-based</td> <td></td> <td></td> <td>Uses alignments of cDNA, EST, or RNA-seq to predict gene structures, including alternative splice events. Can run GMAP and BLAT to do alignment. Can use external gff3 file</td> </tr> <tr> <td>2003</td> <td>EasyGene</td> <td>EasyGene--a prokaryotic gene finder that ranks ORFs by statistical significance. Larsen TS, Krogh A. BMC Bioinformatics. 2003 Jun 3; 4():21.</td> <td>Ab initio</td> <td>HMM, H</td> <td>Prokaryote / Archaea</td> <td></td> </tr> <tr> <td>2003</td> <td>AMIGene (Annotation of MIcrobial Genes)</td> <td>Bocs, S., Cruveiller, S., Vallenet, D., Nuel, G., Medigue, C. 2003 AMIGene: Annotation of MIcrobial Genes Nucleic Acids Res. 31 3723 –3726</td> <td>Ab initio</td> <td>HMM</td> <td>Prokaryote</td> <td></td> </tr> <tr> <td>2003</td> <td>ETOPE</td> <td>Anton Nekrutenko, Wen-Yu Chung, Wen-Hsiung Li. Nucleic Acids Research, Volume 31, Issue 13, 1 July 2003, Pages 3564–3567, https://doi.org/10.1093/nar/gkg597</td> <td>Comparative / evolutionary</td> <td>based on the ratio of non-synonymous to synonymous substitution rates between sequences from different genomes</td> <td>Eukaryote</td> <td>Based on Genscan output. It doesn't predict exons but rather validate exon predicted by other tools.</td> </tr> <tr> <td>2003</td> <td>CRASA</td> <td>A complexity reduction algorithm for analysis and annotation of large genomic sequences. Chuang TJ, Lin WC, Lee HC, Wang CW, Hsiao KL, Wang ZH, Shieh D, Lin SC, Ch'ang LY. Genome Res. 2003 Feb; 13(2):313-22.</td> <td></td> <td></td> <td></td> <td></td> </tr> <tr> <td>2003</td> <td>YACOP</td> <td>Tech M, Merkl R (2003) YACOP: enhanced gene prediction obtained by a combination of existing methods. In Silico Biol 3:441–451</td> <td>combiner: abinitio + evidence</td> <td>Utilizes Glimmer, Critica and ZCURVE</td> <td>Prokaryote / Archaea</td> <td></td> </tr> <tr> <td>2003</td> <td>ZCurve</td> <td>Guo FB, Ou HY, Zhang CT (2003) ZCURVE: a new system for recognizing protein-coding genes in bacterial and archaeal genomes. Nucleic Acids Res 31:1780–1789</td> <td>abinitio</td> <td>Z curve. correlation of dinucleotides.</td> <td>Prokaryote / Archaea</td> <td>Uses the “Z-transform” of DNA as the information source for classification</td> </tr> <tr> <td>2003</td> <td>Eugene'Hom</td> <td>Foissac S, Bardou P, Moisan A, Cros M, Schiex T. EuGene'Hom: a generic similarity-based gene finder using multiple homologous sequences. Nucleic Acids Res 2003; 31: 3742-3745.</td> <td>Evidence-based</td> <td></td> <td>Eukaryote</td> <td></td> </tr> <tr> <td>2004</td> <td>GeneWise</td> <td>GeneWise and Genomewise. Birney E, Clamp M, Durbin R. Genome Res. 2004 May; 14(5):988-95.</td> <td>Hybrid</td> <td></td> <td></td> <td>HMM-based gene prediction tool using extrinsic evidence</td> </tr> <tr> <td>2004</td> <td>Ensembl</td> <td></td> <td>Pipeline Evidence based</td> <td></td> <td></td> <td>Pipeline difficult to install in standalone</td> </tr> <tr> <td>2004</td> <td>RescueNet</td> <td>Mahony S, McInerney JO, Smith TJ, Golden A (2004) Gene prediction using the Self-Organizing Map: automatic generation of multiple gene models. BMC Bioinformatics 5:23</td> <td>Ab initio, evidence</td> <td></td> <td>Prokaryote, Archaea</td> <td>Unsupervised discovery of multiple gene classes using a self-organizing map. No exact start/stop prediction</td> </tr> <tr> <td>2004</td> <td>Reganor</td> <td>McHardy AC, Goesmann A, Puhler A, Meyer F (2004) Development of joint application strategies for two microbial gene finders. Bioinformatics 20:1622–1631</td> <td>combiner: abintito + evidence</td> <td>Uses Glimmer and Critica</td> <td>Prokaryote / Archaea</td> <td></td> </tr> <tr> <td>2004</td> <td>Combiner</td> <td>Allen, J.E., et al. 2004. Computational gene prediction using multiple sources of evidence. Genome Res. 14142–148</td> <td>combiner</td> <td>Linear Combiner that uses a voting function; statistical scoring method that uses decision trees</td> <td></td> <td>Three different algorithms for combining evidence in the Combiner were implemented</td> </tr> <tr> <td>2004</td> <td>GlimmerHMM</td> <td>Majoros, W.H., Pertea, M.,and Salzberg, S.L. TigrScan and GlimmerHMM: two open-source ab initio eukaryotic gene-finders Bioinformatics 2004 2878-2879.</td> <td>Ab initio</td> <td>GHMM</td> <td>eukaryote</td> <td></td> </tr> <tr> <td>2004</td> <td>GeneZilla (formerly "TIGRscan")</td> <td>Majoros, W.H., Pertea, M.,and Salzberg, S.L. TigrScan and GlimmerHMM: two open-source ab initio eukaryotic gene-finders Bioinformatics 2004 2878-2879.</td> <td>Ab initio</td> <td>GHMM</td> <td>eukaryote</td> <td>No longer supported</td> </tr> <tr> <td>2004</td> <td>SNAP (Semi-HMM-based Nucleic Acid Parser)</td> <td>Korf, I. Gene finding in novel genomes. BMC Bioinformatics 5, 59 (2004)</td> <td>Ab initio</td> <td>semi-HMM</td> <td></td> <td></td> </tr> <tr> <td>2004</td> <td>Projector</td> <td>Meyer IM, Durbin R. 2004. Gene structure conservation aids similarity based gene prediction. Nucleic Acids Res. 32:776–83</td> <td>comparative</td> <td>PHMM</td> <td></td> <td>Similar to DOUBLESCAN but extends the model to make use of annotation information on one sequence to inform the other</td> </tr> <tr> <td>2004</td> <td>ExoniPhy</td> <td>Siepel A, Haussler D. 2004. Computational identification of evolutionarily conserved exons. In Proceedings of the Eighth Annual International Conference on Research in Computational Molecular Biology, ed.Gusfield D, Bourne P, Istrail S, Pevzner P, Waterman M, pp. 177–86. New York: Assoc. Comput. Mach.</td> <td>Comparative / evolutionary</td> <td>phylo-HMM</td> <td></td> <td>Phylogenetic HMM that performs ab initio predictions across a multiple-sequence alignment</td> </tr> <tr> <td>2005</td> <td>ExonHunter</td> <td>Bronislava Drejova. Evidence Combination in Hidden Markov Models for Gene Prediction. PhD thesis, the University of Waterloo, 2005. Broii.a Brejova, Daniel G. Brown, Ming Li, and Tomas Vinaf. ExonHunter: a comprehensive approach to gene finding. Bioinformatics, 21 Suppl. 1:i57- i65, 2005.</td> <td>Comparative + evidence driven</td> <td>GHMM</td> <td></td> <td>use genomic sequences, expressed sequence tags and protein databases of related species</td> </tr> <tr> <td>2005</td> <td>JIGSAW</td> <td>Jonathan E. Allen and Steven L. Salzberg. JIGSAW: Integration of Multiple Sources of Evidence for Gene Prediction. Bioinformatics, 21:3596- 3603, 2005.</td> <td>Combiner</td> <td>GHMM-like algorithm</td> <td></td> <td>select the prediction whose structure best represents the consensus</td> </tr> <tr> <td>2005</td> <td>AIR</td> <td>Florea L, Di Francesco V, Miller J, Turner R, Yao A, et al. 2005. Gene and alternative splicing annotation with AIR. Genome Res.</td> <td>evidence</td> <td></td> <td></td> <td>Integrates multiple forms of extrinsic evidence to perform alternative splice junction prediction</td> </tr> <tr> <td>2005</td> <td>GeneMark-ES</td> <td>Lomsadze, A. Gene identification in novel eukaryotic genomes by self-training algorithm. Nucleic Acids Res. 33, 6494–6506 (2005) ; Ter-Hovhannisyan, V., Lomsadze, A., Chernoff, Y. O. &amp; Borodovsky, M. Gene prediction in novel fungal genomes using an ab initio algorithm with unsupervised training. Genome Res. 18, 1979–1990 (2008).</td> <td>Ab initio</td> <td></td> <td>Eukaryote</td> <td></td> </tr> <tr> <td>2005</td> <td>BGF (Beijing Gene Finder)</td> <td>Li, H. et al. Test data sets and evaluation of gene prediction programs on the rice genome. J Comp Sci Tech 20, 446–453 (2005).</td> <td>Ab initio</td> <td>semi HMM</td> <td>Plant (Eukaryote in general?)</td> <td></td> </tr> <tr> <td>2005</td> <td>TWAIN</td> <td>Majoros WH, Pertea M, Salzberg SL. Efficient implementation of a generalized pair hidden Markov model for comparative gene finding. Bioinformatics. 2005;21(9):1782–1788.</td> <td>comparative</td> <td>GPHMM</td> <td></td> <td>Dual genome</td> </tr> <tr> <td>2005</td> <td>GenomeThreader</td> <td>G. Gremme, V. Brendel, M.E. Sparks, and S. Kurtz. Engineering a software tool for gene structure prediction in higher organisms. Information and Software Technology, 47(15):965-978, 2005</td> <td>Evidence based</td> <td>Similarity</td> <td>All</td> <td>The gene structure predictions are calculated using a similarity-based approach where additional cDNA/EST and/or protein sequences are used to predict gene structures via spliced alignments</td> </tr> <tr> <td>2006</td> <td>MaGe</td> <td>Vallenet D, Labarre L, Rouy Z, Barbe V, Bocs S, Cruveiller S, Lajus A, Pascal G, Scarpelli C, Medigue C: MaGe: a microbial genome annotation system supported by synteny results. Nucleic Acids Res. 2006, 34 (1): 53-65. 10.1093/nar/gkj406.</td> <td>Pipeline</td> <td></td> <td>Bacteria</td> <td>AMIGene for protein coding, RBSfinder for ribosome,tRNAscan-SE for tRNA, Rfam for small RNAs and riboswitches,etc. The output is a sort of gff3 (not fully compatible. Define only gene and CDS feature. Gene do not have ID and CDS do not have parent attributes but share locus_tag attribute)</td> </tr> <tr> <td>2006</td> <td>DOGFISH (for ‘detection of genomic features in sequence homologies’)</td> <td>Carter D, Durbin R. 2006. Vertebrate gene finding from multiple-species alignments using a two-level strategy. Genome Biol. 7(Suppl. 1):S6.1–12</td> <td>Comparative</td> <td>HMM</td> <td>vertebrate</td> <td>Two-step program that combines a classifier that scores potential splice sites using a multiple-sequence alignment and an ab initio gene predictor that makes use of the scores from the classifier to predict gene structures. More than two genomes possible.</td> </tr> <tr> <td>2006</td> <td>AUGUSTUS+</td> <td>Stanke M, Schoffmann O, Morgenstern B, Waack S. Gene prediction in eukaryotes with a generalized hidden Markov model that uses hints from external sources. BMC Bioinformatics. 2006;7:62.</td> <td>Hybrid</td> <td>GHMM or CRF</td> <td></td> <td></td> </tr> <tr> <td>2006</td> <td>N-SCAN (a.k.a. TWINSCAN 3.0)</td> <td>Annual International Conference on Research in Computational Molecular Biology RECOMB 2005: Research in Computational Molecular Biology pp 374-388 ; Gross SS, Brent MR. 2006. Using multiple alignments to improve gene prediction. J. Comput. Biol. 13:379–93.</td> <td>Comparative</td> <td></td> <td></td> <td>Can use more than 2 genomes (Extends the TWINSCAN model to N genomes)</td> </tr> <tr> <td>2006</td> <td>ZCURVE_V</td> <td>ZCURVE_V: a new self-training system for recognizing protein-coding genes in viral and phage genomes.</td> <td>ab initio</td> <td>Z curve</td> <td>Virus</td> <td>self-training</td> </tr> <tr> <td>2006</td> <td>TWINSCAN_EST</td> <td>Wei C, Brent MR: Using ESTs to improve the accuracy of de novo gene prediction. BMC Bioinformatics. 2006, 7: 327-10.1186/1471-2105-7-327.</td> <td>Comparative + Evidence</td> <td></td> <td> </td> <td>Two genomes</td> </tr> <tr> <td>2006</td> <td>N_Scan_EST</td> <td>Wei C, Brent MR: Using ESTs to improve the accuracy of de novo gene prediction. BMC Bioinformatics. 2006, 7: 327-10.1186/1471-2105-7-327.</td> <td>Comparative + Evidence</td> <td>HMM</td> <td> </td> <td>HMM-based gene prediction tool that makes use of EST and genomic alignments, incorporating phylogenetic information</td> </tr> <tr> <td>2006</td> <td>Metagene</td> <td>Noguchi, H., Park, J. &amp; Takagi, T. MetaGene: prokaryotic gene finding from environmental genome shotgun sequences. Nucleic Acids Res. 34, 5623–5630 (2006)</td> <td>ab initio</td> <td></td> <td>Metagenomic</td> <td></td> </tr> <tr> <td>2006</td> <td>TiCo</td> <td>An unsupervised classification scheme for improving predictions of prokaryotic TIS. Tech M, Meinicke P. BMC Bioinformatics. 2006 Mar 9; 7():121.</td> <td></td> <td></td> <td>Prokaryote</td> <td>clustering algorithm for completely unsupervised scoring of potential TIS, based on positionally smoothed probability matrices. The algorithm requires an initial gene prediction and the genomic sequence of the organism to perform the reannotation</td> </tr> <tr> <td>2006</td> <td>FGENESH++</td> <td>Solovyev V, Kosarev P, Seledsov I, Vorobyev D. (2006) Automatic annotation of eukaryotic genes, pseudogenes and promoters. Genome Biol. 2006;7 Suppl 1:S10.1-12.</td> <td>Pipeline</td> <td>hybrid: hmm+extrinsinc</td> <td></td> <td>automated version of FGENESH+</td> </tr> <tr> <td>2007</td> <td>Conrad</td> <td>DeCaprio, D. et al. Conrad: gene prediction using conditional random fields. Genome Res. 17, 1389–1398 (2007).</td> <td>comparative</td> <td>semi-Markov conditional random fields (SMCRFs)</td> <td></td> <td>first comparative gene predictor based on SMCRFs. Can use more than 2 genomes</td> </tr> <tr> <td>2007</td> <td>Contrast</td> <td>Gross SS, Do CB, Sirota M, Batzoglou S. 2007. CONTRAST: a discriminative, phylogeny-free approach to multiple informant de novo gene prediction. Genome Biol. 8:R269.</td> <td>Comparative</td> <td>CRF,SVM. Combines local classifiers with the global gene structure model.</td> <td></td> <td>Can also incorporate information from EST alignment. Can use more than 2 genomes. Uses a combination of SVM and CRF predictors, providing a big boost over traditional HMMs</td> </tr> <tr> <td>2007</td> <td>GISMO</td> <td>Krause L, McHardy AC, Nattkemper TW, Pühler A, Stoye J, Meyer F (2007) GISMO—gene identification using a support vector machine for ORF classification. Nucleic Acids Res 35:540–549</td> <td>Abinitio + evidence</td> <td>SVMs</td> <td>Prokaryote</td> <td>Uses SVMs. Model training is based on “reliable” genes found with PFAM protein domain HMMs. Output GFF2</td> </tr> <tr> <td>2007</td> <td>Genomix</td> <td>Coghlan, A. &amp; Durbin, R. Genomix: a method for combining gene-finders' predictions, which uses evolutionary conservation of sequence and intron–exon structure. Bioinformatics 23, 1468–1475 (2007).</td> <td>combiner</td> <td>DP</td> <td>eukaryote</td> <td>use dynamic programming to select the best conserved (top-scoring) predicted exons in the query region, and combine them into a gene structure</td> </tr> <tr> <td>2007</td> <td>GLEAN</td> <td>Elsik, C.G. et al. Creating a honey bee consensus gene set. Genome Biol. 8, R13 (2007).</td> <td>combiner</td> <td>HMM</td> <td>Eukaryote</td> <td>use an unsupervised learning method</td> </tr> <tr> <td>2007</td> <td>FLAN</td> <td>Bao Y, Bolotov P, Dernovoy D, Kiryutin B, Tatusova T. FLAN: a web server for influenza virus genome annotation. Nucleic Acids Res. 2007. pp. W280–284.</td> <td>similarity-based</td> <td></td> <td>Influenza virus</td> <td></td> </tr> <tr> <td>2007</td> <td>transMap</td> <td>Using native and syntenically mapped cDNA alignments to improve de novo gene finding. Stanke M, Diekhans M, Baertsch R, Haussler D. Bioinformatics. 2008 Mar 1; 24(5):637-44. Zhu J, Sanborn JZ, Diekhans M, Lowe CB, Pringle TH, Haussler D. 2007. Comparative genomics search for losses of long-established genes on the human lineage. PLOS Comput. Biol. 3:e247.</td> <td>Evidence</td> <td></td> <td>Eukaryote</td> <td>Uses whole-genome alignments to project existing annotations from one genome to one or more other genomes. first developed in conjunction with improvements to AUGUSTUS to model extrinsic information</td> </tr> <tr> <td>2007</td> <td>GLIMMER3</td> <td>Delcher AL, et al. Identifying bacterial genes and endosymbiont DNA with Glimmer, Bioinformatics , 2007, vol. 23 (pg. 673-679)</td> <td>Abinitio</td> <td>IMM</td> <td>bacteria, archæa and viruses</td> <td>It integrates Ribosome binding sites evidence directly into the gene-finding algorithm. It distinguishs host and endosymbiont DNA.</td> </tr> <tr> <td>2008</td> <td>SCGPred</td> <td>SCGPred: a score-based method for gene structure prediction by combining multiple sources of evidence. Li X, Ren Q, Weng Y, Cai H, Zhu Y, Zhang Y Genomics Proteomics Bioinformatics. 2008 Dec; 6(3-4):175-85.</td> <td>Combiner</td> <td></td> <td>Eukaryote</td> <td>automated eukaryotic gene structure annonator that computes weighted consensus gene structure based on multiple sources of available evidence</td> </tr> <tr> <td>2008</td> <td>RAST</td> <td>Aziz RK, Bartels D, Best AA, DeJongh M, Disz T, Edwards RA, Formsma K, Gerdes S, Glass EM, Kubal M, et al. The RAST Server: rapid annotations using subsystems technology, BMC Genomics , 2008, vol. 9 pg. 75</td> <td>pipeline</td> <td></td> <td>bacterial and archaeal</td> <td>Online service that identifies protein-encoding, rRNA and tRNA genes, assigns functions to the genes, predicts which subsystems are represented in the genome, uses this information to reconstruct the metabolic network and makes the output easily downloadable for the user.</td> </tr> <tr> <td>2008</td> <td>Maker</td> <td>Cantarel, B. L. et al. Maker. Genome Res. 18, 188–96 (2008).</td> <td>Combiner</td> <td></td> <td></td> <td>It uses proteins, transcripts ... Abinitio: Augustus, Fgnesh,Genemark,snap</td> </tr> <tr> <td>2008</td> <td>Evigan</td> <td>Liu, Q., Mackey, A. J., Roos, D. S. &amp; Pereira, F. C. N. Evigan: A hidden variable model for integrating gene evidence for eukaryotic gene prediction. Bioinformatics 24, 597–605 (2008).</td> <td>Combiner</td> <td>Dynamic Bayes networks (DBNs)</td> <td>Eukaryote</td> <td>Choose the best possible set of exons and combine them in a gene model. Weight of different sources. Unsupervised learning method</td> </tr> <tr> <td>2008</td> <td></td> <td>Y. Zhou, Y. Liang, C. Hu, L. Wang, X. Shi, An artificial neural network method for combining gene prediction based on equitable weights, NeuroComputing 71 (2007) 538–543</td> <td>combiner</td> <td>RBFN</td> <td></td> <td></td> </tr> <tr> <td>2008</td> <td>Evidence Modeler (EVM)</td> <td>Haas, B. J. et al. Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments. Genome Biol. 9, R7 (2008).</td> <td>Combiner</td> <td></td> <td></td> <td>choose the best possible set of exons and combine them in a gene model weight of different sources. Evidence based chooser.</td> </tr> <tr> <td>2008</td> <td>Chemgenome2.0</td> <td>Poonam Singhal, B. Jayaram, Surjit B. Dixit and David L. Beveridge. Prokaryotic Gene Finding based on Physicochemical Characteristics of Codons Calculated from Molecular Dynamics Simulations.Biophysical Journal,2008,Volume:94 Issue:11, 4173-4183 ]</td> <td>Ab initio</td> <td></td> <td>Procaryote</td> <td>Prokaryotic Gene Finding Based on Physicochemical Characteristics of Codons Calculated from Molecular Dynamics Simulations</td> </tr> <tr> <td>2008</td> <td>MetaGeneAnnotator (MGA)</td> <td>Noguchi H, Taniguchi T, Itoh T (2008) Meta- GeneAnnotator: detecting species-specific patterns of ribosomal binding site for precise gene prediction in anonymous prokaryotic and phage genomes. DNA Res 15:387–396.</td> <td>abinitio</td> <td></td> <td>Prokaryote</td> <td>MGA is a self-training gene prediction tool for all kinds of prokaryotic genes including atypical genes such as horizontally transferred and prophage-encoded genes</td> </tr> <tr> <td>2009</td> <td>mGene</td> <td>Schweikert, G. et al. mGene: accurate SVM-based gene finding with an application to nematode genomes. Genome Res. 19, 2133–43 (2009).</td> <td>Ab initio</td> <td>Structural HMM combined with discrimination training techniques similar to SVMs</td> <td></td> <td>No longer supported</td> </tr> <tr> <td>2009</td> <td>Orphelia</td> <td></td> <td>Ab initio</td> <td>Neural network</td> <td>Metagenomic</td> <td></td> </tr> <tr> <td>2009</td> <td>MiGAP (Microbial Genome Annotation Pipeline )</td> <td>Sugawara H. et al. (2009) Microbial genome annotation pipeline (MiGAP) for diverse users. In: Proceedings of the 20th International Conference on Genome Informatics, Yokohama, Japan, S–001–1–2.</td> <td>Pipeline: MetaGeneAnnotator + tRNAscan-SE + rRNA db</td> <td></td> <td>Prokaryote</td> <td></td> </tr> <tr> <td>2009</td> <td>DAWGPAWS</td> <td>Estill, J. C. &amp; Bennetzen, J. L. The DAWGPAWS pipeline for the annotation of genes and transposable elements in plant genomes. Plant Methods 5, 1–11 (2009).</td> <td></td> <td>Eukaryote / Plant</td> <td></td> <td>pipeline for the annotation of genes and transposable elements in plant genomes</td> </tr> <tr> <td>2010</td> <td>MetaGeneMark</td> <td>Zhu, W., Lomsadze, A. &amp; Borodovsky, M. Ab initio gene identification in metagenomic sequences. Nucleic Acids Res. 38, 1–15 (2010).</td> <td>Ab initio</td> <td>HMM</td> <td>Metagenome</td> <td>Self training</td> </tr> <tr> <td>2010</td> <td>Prodigal (PROkaryotic DYnamic programming Gene-finding ALgorithm)</td> <td>Hyatt, D. et al. Prodigal: prokaryotic gene recognition and translation initiation site identification. BMC Bioinformatics 11, 119 (2010)</td> <td>Ab initio</td> <td>dynamic programming + HMM. Log-likelihood coding statistics trained from data.</td> <td>Prokaryote, Metagenome</td> <td>Self training</td> </tr> <tr> <td>2010</td> <td>GenePRIMP</td> <td>Pati A. Ivanova N.N. Mikhailova N. Ovchinnikova G. Hooper S.D. Lykidis A. Kyrpides N.C. GenePRIMP: a gene prediction improvement pipeline for prokaryotic genomes Nat. Methods 2010 7 455457</td> <td>-</td> <td>-</td> <td>Prokaryote</td> <td>evidence-based evaluation</td> </tr> <tr> <td>2010</td> <td>VIGOR (Viral Genome ORF Reader)</td> <td>Wang S, Sundaram JP, Spiro D. 2010. VIGOR, an annotation program for small viral genomes. BMC Bioinformatics 11:451. http://dx.doi.org /10.1186/1471-2105-11-451.</td> <td>Evidence</td> <td></td> <td>Virus: influenza virus, rotavirus, rhinovirus and coronavirus subtypes</td> <td>Web application tool</td> </tr> <tr> <td>2010</td> <td>FragGeneScan</td> <td>Rho M, Tang H, Ye Y (2010) FragGeneScan: predicting genes in short and error-prone reads. Nucleic Acids Res 38:e191</td> <td>ab initio</td> <td>HMM</td> <td>Metagenome</td> <td>HMM-based. Combines sequencing error models with codon usage</td> </tr> <tr> <td>2010</td> <td>MetaGeneMark</td> <td>Zhu W, Lomsadze A, Borodovsky M (2010) Ab initio gene identification in metagenomic sequences. Nucleic Acids Res 38:e132</td> <td>abinitio</td> <td></td> <td>Metagenome</td> <td>Update of GeneMark.hmm with improved model parameters for metagenomic samples</td> </tr> <tr> <td>2010</td> <td>Gnomon</td> <td>Souvorov, A. et al. Gnomon — the NCBI eukaryotic gene prediction tool. National Center for Biotechnology Information, (2010).</td> <td>Abinitio</td> <td>HMM; Translational and splice signals are described using WMM and WAM models</td> <td></td> <td>Following the Genscan logic Gnomon recognizes as HMM states coding exons and introns on both strands and intergenic sequence</td> </tr> <tr> <td>2011</td> <td>MAKER2</td> <td>Holt, C. &amp; Yandell, M. MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects. BMC Bioinformatics 12, 491 (2011).</td> <td>Pipeline / combiner</td> <td>Evidence or abinitio or abinitio evidence driven</td> <td>Eukaryote / Prokaryote</td> <td>It uses proteins, transcripts ... Abinitio: Augustus, Fgnesh,Genemark,snap</td> </tr> <tr> <td>2011</td> <td>GenSAS</td> <td>Lee T, Peace C, Jung S, Zheng P, Main D, Cho I (2011) GenSAS: an online integrated genome sequence annotation pipeline. In: 4th International conference on biomedical engineering and informatics (BMEI), Shanghai, 2011, pp. 1967–1973. doi: 10.1109/BMEI.2011.6098712</td> <td>pipeline</td> <td></td> <td></td> <td>An online integrated genome sequence annotation pipeline</td> </tr> <tr> <td>2011</td> <td>VMGAP (TheViral MetaGenome Annotation Pipeline)</td> <td>Lorenzi, H. A. et al. TheViral MetaGenome Annotation Pipeline(VMGAP): an automated tool for the functional annotation of viral metagenomic shotgun sequencing data. Stand. Genomic Sci. 4, 418–429 (2011).</td> <td>Pipeline</td> <td></td> <td>Viruses</td> <td></td> </tr> <tr> <td>2012</td> <td>eCRAIG (ensemble CRAIG)</td> <td>Bernal A, Crammer K, Pereira F: Automated gene-model curation using global discriminative learning. Bioinformatics. 2012, 28 (12): 1571-1578. 10.1093/bioinformatics/bts176.</td> <td>combiner</td> <td>CRF-based</td> <td></td> <td></td> </tr> <tr> <td>2012</td> <td>MOCAT</td> <td>Kultima JR, Sunagawa S, Li J, Chen W, Chen H, Mende DR, Arumugam M, Pan Q, Liu B, Qin J (2012) MOCAT: a metagenomics assembly and gene prediction toolkit. PLoS One 7:e47656</td> <td>pipeline</td> <td>Use Prodigal or MetaGeneMark</td> <td>Metagenome</td> <td></td> </tr> <tr> <td>2013</td> <td>GIIRA (Gene Identification Incorporating RNA-Seq data and Ambiguous reads)</td> <td>Zickmann F, Lindner MS, Renard BY (2013) GIIRA–RNA-Seq driven gene finding incorporating ambiguous reads. Bioinformatics 30:606–613</td> <td>abinitio evidence driven</td> <td>maximum-flow approach</td> <td>Eukaryote, Prokaryote</td> <td>Based on the observed mapping coverage, GIIRA identifies candidate genes that are refined in further validating steps.</td> </tr> <tr> <td>2013</td> <td>Eugene-P</td> <td>Next-generation Annotation of Prokaryotic Genomes with EuGene-P: Application to Sinorhizobium meliloti 2011. E. Sallet et al. DNA Res. 2013</td> <td></td> <td></td> <td>Prokaryote</td> <td></td> </tr> <tr> <td>2013</td> <td>MetaGUN</td> <td>Liu Y, Guo J, Hu G, Zhu H (2013) Gene prediction in metagenomic fragments based on the SVM algorithm. BMC Bioinformatics 14:S12</td> <td>abinitio</td> <td>SVM-based. Phylogenetic binning and assignment of protein sequences to each bin</td> <td>Metagenome</td> <td></td> </tr> <tr> <td>2014</td> <td>ZUPLS</td> <td>Song, K., Tong, T., and Wu, F. (2014). Predicting essential genes in prokaryotic genomes using a linear method: ZUPLS. Integr. Biol. 6, 460–469. doi: 10.1039/c3ib40241j</td> <td>ab initio</td> <td>Z-curve</td> <td>Prokaryote</td> <td></td> </tr> <tr> <td>2014</td> <td>OMIGA (Optimized Maker-Based Insect Genome Annotation)</td> <td>Liu J. Xiao H. Huang S. Li F. OMIGA: Optimized Maker-Based Insect Genome Annotation Mol. Genet. Genomics 2014 289 567 573</td> <td>pipeline (MAKER)</td> <td>Augustus,Snap,GeneMark</td> <td>Insect</td> <td></td> </tr> <tr> <td>2014</td> <td>GeneMark-ET</td> <td>Lomsadze, A., Burns, P. D. &amp; Borodovsky, M. Integration of mapped RNA-Seq reads into automatic training of eukaryotic gene finding algorithm. 42, 1–8 (2014).</td> <td>Ab initio</td> <td>HMM</td> <td>Eukaryote</td> <td>Self training</td> </tr> <tr> <td>2014</td> <td><a href=https://github.com/tseemann/prokka>Prokka</a></td> <td>Seemann T., Prokka: rapid prokaryotic genome annotation. Bioinformatics 2014 Jul 15;30(14):2068-9. PMID:24642063</td> <td>pipeline</td> <td>Ab initio + evidence-based for functional annotation</td> <td>prokaryote</td> <td>Do structural and functional annotation</td> </tr> <tr> <td>2014</td> <td>DFAST (DDBJ Fast Annotation and Submission Tool)</td> <td>Seemann T. (2014) Prokka: rapid prokaryotic genome annotation. Bioinformatics , 30, 2068–2069. DFAST: a flexible prokaryotic genome annotation pipeline for faster genome publication. Yasuhiro Tanizawa, Takatomo Fujisawa, Yasukazu Nakamura. Bioinformatics, Volume 34, Issue 6, 15 March 2018, Pages 1037–1039</td> <td>Pipeline</td> <td></td> <td>Prokaryote</td> <td>The original version of DFAST employs the lightweight command-line program Prokka as an annotation engine. Now DFAST uses MetaGeneAnnotator (MGA) by default to predict CDSs and GHOSTX as a default aligner. Standalone and web versions. Data ready to submit to DDBJ</td> </tr> <tr> <td>2015</td> <td>Ipred</td> <td>Zickmann, F. &amp; Renard, B. Y. IPred - integrating ab initio and evidence based gene predictions to improve prediction accuracy. BMC Genomics 16, 134 (2015).</td> <td>Combiner evidence-based</td> <td></td> <td></td> <td>choose the best possible set of exons and combine them in a gene model. Evidence based chooser. Can also model gene form evidence only.</td> </tr> <tr> <td>2015</td> <td>GASS ( Genome Annotation based on Species Similarity)</td> <td>GASS: genome structural annotation for Eukaryotes based on species similarity. Wang Y, Chen L, Song N, Lei X. BMC Genomics. 2015 Mar 4; 16():150.</td> <td>comparative</td> <td>shortest path model and DP</td> <td></td> <td></td> </tr> <tr> <td>2016</td> <td>BRAKER1</td> <td>Lange, S., Hof, K., Lomsadze, A., Borodovsky, M. &amp; Stanke, M. BRAKER1 : Unsupervised RNA-Seq-Based Genome Annotation with GeneMark-ET and AUGUSTUS. 24, 2014 (2014).</td> <td>Hybrid / Pipeline</td> <td></td> <td></td> <td>Pipeline: GeneMark-ET + Augustus</td> </tr> <tr> <td>2016</td> <td>Companion</td> <td>Steinbiss, S. et al. Companion: a web server for annotation and analysis of parasite genomes. Nucleic Acids Res. 44, W29–W34 (2016)</td> <td></td> <td></td> <td></td> <td>Pipeline for automatic eukaryotic parasite annotation</td> </tr> <tr> <td>2016</td> <td>Gmove</td> <td>Dubarry, M. et al. Gmove a tool for eukaryotic gene predictions using various evidences. F1000Reserach 34, 2011 (2016).</td> <td></td> <td></td> <td>Eukaryote</td> <td></td> </tr> <tr> <td>2016</td> <td>AugustusCGP</td> <td>König S, Romoth LW, Gerischer L, Stanke M. Bioinformatics. 2016 Nov 15; 32(22):3388-3395.</td> <td>comparative</td> <td></td> <td>Eukaryote</td> <td>mutiple genomes</td> </tr> <tr> <td>2016</td> <td>CAT Comparative Analysis toolkit</td> <td>Fiddes, I. T. et al. Comparative Annotation Toolkit (CAT)—simultaneous clade and personal genome annotation. Genome Res. https://doi.org/10.1101/gr.233460.117 (2018).</td> <td>pipeline</td> <td>Evidence based, comparative-abinitio (AugustusCGP)</td> <td></td> <td>takes as input a HAL-format multiple whole genome alignment. Output GFF3 + many plots</td> </tr> <tr> <td>2016</td> <td>CESAR</td> <td>Coding exon-structure aware realigner (CESAR) utilizes genome alignments for accurate comparative gene annotation. Sharma V, Elghafari A, Hiller M. Nucleic Acids Res. 2016 Jun 20; 44(11):e103.</td> <td>comparative</td> <td></td> <td></td> <td>Uses a HMM to adjust splice sites in whole-genome alignments, improving transcript projections</td> </tr> <tr> <td>2016</td> <td>PGAP</td> <td>Tatusova T. et al. (2016) NCBI prokaryotic genome annotation pipeline. Nucleic Acids Res ., 44, 6614–6624.</td> <td>pipeline</td> <td>GenemarkS+ Glimmer + extrinsec data</td> <td>Prokaryote</td> <td>This is the NCBI annotation service incorporated in its submission system, but it is only available for GenBank submitters.</td> </tr> <tr> <td>2017</td> <td> GeMoMa</td> <td>Keilwagen, J., Hartung, F., Paulini, M., Twardziok, S. O. &amp; Grau, J. Combining RNA-seq data and homology-based gene prediction for plants , animals and fungi. (2017).</td> <td></td> <td></td> <td></td> <td>homology-based gene prediction program</td> </tr> <tr> <td>2017</td> <td> funannotate</td> <td>doi.org/10.5281/zenodo.2576527</td> <td>Pipeline</td> <td>Evidence Modeler + Augustus + GeneMark-ES/ET + evidence + PASA</td> <td>built specifically for fungi, but will also work with higher eukaryotes</td> <td>homology-based gene prediction program</td> </tr> <tr> <td>2017</td> <td>GAWN</td> <td>unpublished - https://github.com/enormandeau/gawn</td> <td>pipeline evidence-based only</td> <td>GMAP to create gene and cufflinks and TransDecoder to add UTR</td> <td>eukaryote</td> <td></td> </tr> <tr> <td>2018</td> <td>FunGAP</td> <td>Min B, Grigoriev IV, Choi IG. FunGAP: Fungal Genome Annotation Pipeline using evidence-based gene model evaluation. Bioinformatics (Oxford, England). 2017;33(18):2936–7.</td> <td>pipeline</td> <td></td> <td></td> <td></td> </tr> <tr> <td>2018</td> <td>BRAKER2</td> <td>BRAKER2: automatic eukaryotic genome annotation with GeneMark-EP+ and AUGUSTUS supported by a protein database - https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7787252/</td> <td></td> <td>Hybrid</td> <td>eukaryote</td> <td></td> </tr> <tr> <td>2018</td> <td>G-OnRamp</td> <td></td> <td>Ab-initio Web-based</td> <td>Augustus,GlimmerHMM,SNAP</td> <td></td> <td></td> </tr> <tr> <td>2018</td> <td>VIRULIGN</td> <td>VIRULIGN: fast codon-correct alignment and annotation of viral genomes. Pieter J K Libin, Koen Deforche, Ana B Abecasis, Kristof Theys. Bioinformatics, Volume 35, Issue 10, 15 May 2019, Pages 1763–1765, https://doi.org/10.1093/bioinformatics/bty851</td> <td>Similarity</td> <td></td> <td>Virus</td> <td></td> </tr> <tr> <td>2019</td> <td>GAAP</td> <td>Jinhwa Kong, Sun Huh, Jung-Im Won, Jeehee Yoon, Baeksop Kim, and Kiyong Kim. GAAP: A Genome Assembly + Annotation Pipeline. BioMed Research International, Volume 2019, Article ID 4767354, 12 pages</td> <td>pipeline</td> <td>Augustus,EVM,MAKER,PASA</td> <td></td> <td>Genome Assembly + Annotation Pipeline</td> </tr> <tr> <td>2019</td> <td>VAPiD (Viral Annotation Pipeline and iDentification)</td> <td>Ryan C. Shean, Negar Makhsous, Graham D. Stoddard, Michelle J. Lin &amp; Alexander L. Greninger. VAPiD: a lightweight cross-platform viral annotation pipeline and identification tool to facilitate virus genome submissions to NCBI GenBank. BMC Bioinformaticsvolume 20, Article number: 48 (2019)</td> <td>pipeline</td> <td></td> <td>Virus</td> <td></td> </tr> <tr> <td>2019</td> <td>Vgas (Viral Genome Annotation System)</td> <td>Kai-Yue Zhang, Yi-Zhou Gao, Meng-Ze Du, Shuo Liu, Chuan Dong, and Feng-Biao Guo. Vgas: A Viral Genome Annotation System. Front Microbiol. 2019; 10: 184.</td> <td>abinitio + similarity-based</td> <td>ZCURVE_V + BLASTp</td> <td>Virus</td> <td>In their paper they say: When combining Vgas with GeneMarkS and Prodigal, better prediction results could be obtained than with each of the three individual programs.</td> </tr> <tr> <td>2020</td> <td>VADR</td> <td>VADR: validation and annotation of virus sequence submissions to GenBank. Alejandro A. Schäffer, Eneida L. Hatcher, Linda Yankie, Lara Shonkwiler, J. Rodney Brister, Ilene Karsch-Mizrachi &amp; Eric P. Nawrocki. BMC Bioinformatics 21, 211 (2020). https://doi.org/10.1186/s12859-020-3537-3</td> <td>HMM + similarity</td> <td></td> <td>Virus</td> <td></td> </tr> <tr> <td>2020</td> <td><a href=https://github.com/ikmb-denbi/genome-annotation>esga</a></td> <td></td> <td>Nextflow pipeline focused on the use of Augustus</td> <td>De-novo with evidence based</td> <td>Eukaryote</td> <td>effort made by Marc Höppner to get the best of Augustus</td> </tr> <tr> <td>2020</td> <td>Helixer</td> <td>Helixer: cross-species gene annotation of large eukaryotic genomes using deep learning - https://doi.org/10.1093/bioinformatics/btaa1044</td> <td>deep learning</td> <td></td> <td>Eukaryote</td> <td>cross-species annotation</td> </tr> <tr> <td>2021</td> <td>Bakta</td> <td>Rapid &amp; standardized annotation of bacterial genomes &amp; plasmids</td> <td>pipeline</td> <td></td> <td>Bacteria + plasmid</td> <td>It outpout among others embl files</td> </tr> <tr> <td>2021</td> <td>FINDER</td> <td>FINDER: an automated software package to annotate eukaryotic genes from RNA-Seq data and associated protein sequences. https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-04120-9</td> <td>Pipeline</td> <td>Protein + RNAseq alignement + Abinitio (BRAKER)</td> <td>Eukaryote</td> <td>It output GTF files</td> </tr> <tr> <td>2022</td> <td>Pharokka</td> <td>Pharokka: a fast scalable bacteriophage annotation tool - https://doi.org/10.1093/bioinformatics/btac776</td> <td>Pipeline</td> <td>prodigal is used for CDS</td> <td>bacteriophage</td> <td></td> </tr> <tr> <td>2023</td> <td>FLAG</td> <td>FLAG: Find, Label, Annotate Genomes, a fully automated tool for genome gene structural and functional annotation of highly fragmented non-model species https://www.biorxiv.org/content/10.1101/2023.07.14.548907v1.full</td> <td>Pipeline</td> <td>Protein + RNAseq alignement + Abinitio</td> <td>Eukaryote</td> <td>Nextflow pipeline very complete</td> </tr> <tr> <td>2023</td> <td>BRAKER3</td> <td>BRAKER3: Fully automated genome annotation using RNA-Seq and protein evidence with GeneMark-ETP, AUGUSTUS and TSEBRA - https://pubmed.ncbi.nlm.nih.gov/37398387/</td> <td></td> <td>Hybrid</td> <td>eukaryote</td> <td></td> </tr> <tr> <td>2023</td> <td>Galba</td> <td>Galba: genome annotation with miniprot and AUGUSTUS - https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-023-05449-z</td> <td>pipeline</td> <td>Hybrid = Ab-intio evidence driven</td> <td>eukaryote</td> <td>use Augustus + miniprot</td> </tr> <tr> <td>2024</td> <td>Tiberius</td> <td>Tiberius: end-to-end deep learning with an HMM for gene prediction - https://academic.oup.com/bioinformatics/article/40/12/btae685/7903281</td> <td>deep learning</td> <td></td> <td>eukaryote</td> <td></td> </tr> </tbody> </table> <hr> <p><strong>Legend:</strong> </p> <table> <thead> <tr> <th>Term</th> <th>Description</th> </tr> </thead> <tbody> <tr> <td>Hybrid</td> <td>ab initio and evidence based = HMM-based gene prediction tool using extrinsic evidence</td> </tr> <tr> <td>Comparative</td> <td>genome sequence comparison</td> </tr> <tr> <td>CHMM</td> <td>class HMM</td> </tr> <tr> <td>CRF</td> <td>conditional random field; HMM</td> </tr> <tr> <td> DBN</td> <td>Dynamic Bayes network</td> </tr> <tr> <td>DP</td> <td>dynamic programming</td> </tr> <tr> <td> EHMM</td> <td>evolutionary HMM</td> </tr> <tr> <td>GHMM</td> <td>generalized HMM</td> </tr> <tr> <td>GPHMM</td> <td>generalized pair HMM</td> </tr> <tr> <td>HMM</td> <td>hidden Markov model</td> </tr> <tr> <td>IMM</td> <td>Interpolated Markov model</td> </tr> <tr> <td>LDA</td> <td>Linear Discriminant Analysis</td> </tr> <tr> <td>MDD</td> <td>maximal dependence decomposition</td> </tr> <tr> <td>ML</td> <td>maximum likelihood</td> </tr> <tr> <td>MM</td> <td>Markov Model</td> </tr> <tr> <td>NN</td> <td>Neural Networks</td> </tr> <tr> <td>PHMM</td> <td>pair HMM</td> </tr> <tr> <td>phyloHMM</td> <td>phylogenetic HMM</td> </tr> <tr> <td>RBFN</td> <td>Radial Basis Function Network</td> </tr> <tr> <td>SVM</td> <td>support vector machine</td> </tr> <tr> <td>WAM</td> <td>weight array matrix</td> </tr> </tbody> </table> <hr> <p><strong>Interesting publications</strong><br> <a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC311133/#B39>Rogic, S., Mackworth, A. K., &amp; Ouellette, F. B. (2001). Evaluation of gene-finding programs on mammalian sequences. Genome research, 11(5), 817-32.</a><br> <a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3511556/ >Goodswen, S. J., Kennedy, P. J., &amp; Ellis, J. T. (2012). Evaluating high-throughput ab initio gene finders to discover proteins encoded in eukaryotic pathogen genomes missed by laboratory techniques. PloS one, 7(11), e50609.</a><br> <a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5655831/ >Chowdhury, B., Garai, A., &amp; Garai, G. (2017). An optimized approach for annotation of large eukaryotic genomic sequences using genetic algorithm. BMC bioinformatics, 18(1), 460. doi:10.1186/s12859-017-1874-7</a> <a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6450745/ >Joel Armstrong, Ian T. Fiddes, Mark Diekhans and Benedict Paten. Whole-Genome Alignment and Comparative Annotation.Annu Rev Anim Biosci. 2019 Feb 15; 7: 41–64.</a><br> Alice Carolyn McHardy Andreas Kloetgen. Finding Genes in Genome Sequence. Bioinformatics pp 271-291<br> <a href=http://www.pfb.info.pl/files/kwartalnik/3_2005/Baczkowski-Mackiewicz.pdf>Bączkowski, K., Mackiewicz, K., Kowalczuk, M., Banaszak, J. and Cebrat, S., “Od sekwencji do funkcji– poszukiwanie genów i ich adnotacje,” Biotechnologia 3(70), 22–44 (2005)</a> </p> <p><strong>Interesting books</strong><br> Principles of Gene Manipulation and Genomics. De Sandy B. Primrose, Richard Twyman </p> </article> </div> <script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script> </div> <button type=button class="md-top md-icon" data-md-component=top hidden> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8z"/></svg> Back to top </button> </main> <footer class=md-footer> <nav class="md-footer__inner md-grid" aria-label=Footer> <a href=../../.. class="md-footer__link md-footer__link--prev" aria-label="Previous: Awesome Genome Annotation Tools"> <div class="md-footer__button md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg> </div> <div class=md-footer__title> <span class=md-footer__direction> Previous </span> <div class=md-ellipsis> Awesome Genome Annotation Tools </div> </div> </a> <a href=../annotation_tools_mitome/ class="md-footer__link md-footer__link--next" aria-label="Next: Mitochondrion"> <div class=md-footer__title> <span class=md-footer__direction> Next </span> <div class=md-ellipsis> Mitochondrion </div> </div> <div class="md-footer__button md-icon"> <svg xmlns=http://www.w3.org/2000/svg viewbox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11z"/></svg> </div> </a> </nav> <div class="md-footer-meta md-typeset"> <div class="md-footer-meta__inner md-grid"> <div class=md-copyright> Made with <a href=https://squidfunk.github.io/mkdocs-material/ target=_blank rel=noopener> Material for MkDocs </a> </div> </div> </div> </footer> </div> <div class=md-dialog data-md-component=dialog> <div class="md-dialog__inner md-typeset"></div> </div> <script id=__config type=application/json>{"base": "../../..", "features": ["announce.dismiss", "content.action.edit", "content.action.view", "content.code.annotate", "content.code.copy", "content.tooltips", "header.autohide", "navigation.footer", "navigation.indexes", "navigation.tabs", "navigation.top", "navigation.tracking", "search.highlight", "search.share", "search.suggest", "toc.follow", "toc.integrate"], "search": "../../../assets/javascripts/workers/search.d50fe291.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script> <script src=../../../assets/javascripts/bundle.56ea9cef.min.js></script> <script src=https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js></script> <script src=../../../javascripts/tablesort.js></script> </body> </html>